extensions/wikia/AdEngine/js/lookup/prebid/prebidHelper.js

```diff
@@ -0,0 +1,34 @@
+define('ext.wikia.adEngine.lookup.prebid.prebidHelper', function() {
+	var adUnits = [];
+
+	function getAdUnits(adapter, slots) {
+		var adaptersAdUnits = [];
+
+		Object.keys(slots).forEach(function(slotName) {
+			adaptersAdUnits.push(adapter.prepareAdUnit(slotName, slots[slotName]));
+		});
+
+		return adaptersAdUnits;
+	}
+
+	function addAdUnits(adapterAdUnits) {
+		adapterAdUnits.forEach(function (adUnit) {
+			adUnits.push(adUnit);
+		});
+	}
+
+	function setupAdUnits(adapters, skin) {
```

Comment:
this method does set up and then returns addUnits, `getAllAdUnits` and `setupAdUnits` partially reflects what this method does.